assets/js/upload.js

```diff
@@ -171,9 +171,121 @@ function setupImageUpload() {
     window.removeEventListener('beforeunload', beforeUnload);
   }
 
+  function createTagError(message) {
+    const buttonAfter = $('#tagsinput-save');
+    const errorElement = makeEl('span', { className: 'help-block tag-error' });
+
+    errorElement.innerText = message;
```

Comment:
I believe these can actually also be combined
```suggestion
    const errorElement = makeEl('span', { className: 'help-block tag-error', innerText: message });
```